runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -110,6 +110,16 @@ template <typename LoaderDomain> class BaseLoader
   void loadStridesAndPaddings(Param &param, const OptionsType *options);
   // Load Pool2D param
   template <typename Param> void loadPool2DOptions(Param &param, const Pool2DOptions *options);
+  // Get BuiltinOperator
+  BuiltinOperator getBuiltinOperator(const Operator *op)
+  {
+    auto const builtin_opcode = _model->operator_codes()->Get(op->opcode_index());
+    auto builtin_op = builtin_opcode->builtin_code();
+    if (builtin_op < BuiltinOperator::BuiltinOperator_PLACEHOLDER_FOR_GREATER_OP_CODES)
```

Comment:
Does it work for existing circles? `PLACEHOLDER_FOR_GREATER_OP_CODES` is 127. Existing circle has the builtin_op as 254, 254, ... for extended op in type of uint8_t.

Reply:
Existing circles also parsed by this loader, and `builtin_opcode->builtin_code()` return `0` because existing circles don't have new `builtin_code` field. (old `builtin_code` field is parsed as `deprecated_builtin_code` field)